tests/unittests/config_test.py

```diff
@@ -0,0 +1,14 @@
+from nav.config import _config_resource_walk
+
+
+from unittest import TestCase
+
+
+class TestConfigResourceWalk(TestCase):
+    def test___config_resource_walk(self):
+        # result should be many, many relative paths as strings
```

Comment:
If I see this test failing in the test report, I still have to read the test code to understand what requirement is failing.  I feel like this should be part of the test method name rather than being a comment that explains the test.

E.g. `test_should_return_list_of_relative_pathnames()` would look like "`TestConfigResourceWalk.test_should_return_list_of_relative_pathnames` failed" in the report.